Formula/ca-certificates.rb

```diff
@@ -69,7 +69,9 @@ def macos_post_install
         -l -L
         -c #{tmpfile.path}
       ]
-      verify_args << "-R" << "offline" if MacOS.version >= :high_sierra
+      on_high_sierra :or_newer do
+        verify_args << "-R" << "offline"
+      end
```

Comment:
This isn't in the class root -- do we need to change this?

Reply:
Currently, we take an opt-out approach to which non-root-level nodes can't use `if` statements. This allows `caveats` and all other DSL components except `install`, `post_install`, `service` and `test` to have the `on_{system}` blocks. Instead, it's probably worth an opt-in approach where we only allow it in `caveats` and the other places where it's necessary for what's being generated for the JSON.